etna/packages/etna-js/package.json

```diff
@@ -3,14 +3,19 @@
   "version": "1.1.1-ui",
   "description": "Shared JS modules for Mount Etna.",
   "scripts": {
-    "test": "tsc && jest"
+    "test": "tsc && jest",
+    "build": "find . -name \"*.ts\" -not -path \"*/node_modules/*\" -print0 | xargs -0 -I {} tsc --downlevelIteration --lib es2019 --skipLibCheck {}"
   },
   "author": "",
   "license": "GPL-2.0-only",
   "dependencies": {
     "animate.css": "^4.1.0",
+    "hyperformula": "^2.0.0",
     "react-notifications-component": "^2.4.0"
   },
+  "bundledDependencies": [
+    "hyperformula"
```

Comment:
This seems to make the other apps not break, even though I've introduced a new `etna-js` dependency ?? :shrug: Without this, non-affected apps like Metis would break upon webpack build because they have to include all `etna-js` dependencies, even if they don't use them directly, so we had to add the new dependencies to every application. However, I think there is some [side effect](https://stackoverflow.com/a/25044361) when used with local packages, where these dependencies are then included or found? They did automatically propagate to the `package-lock.json` files in each app, without me having to add them to `package.json` manually, which seems like a plus...

Reply:
Either this line or explicitly calling it out in line 13 above...

Reply:
Yes, but I need to warn you -- my experience is that each npm release __changes its mind about how this works__ so we may see issues in the future.  The only real solution which is beyond our scope is to move towards that single npm package approach.

Reply:
Well phooey... okay, so I guess we'll get bitten by this at some point and revisit ... :cry: